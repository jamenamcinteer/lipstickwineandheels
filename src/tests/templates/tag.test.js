import React from 'react'
import { shallow } from 'enzyme'
import TagTemplate from '../../templates/tag'
import data from '../fixtures/data'

test('should render tag template correctly', () => {
  const wrapper = shallow(<TagTemplate data={data.data} />)
  expect(wrapper).toMatchSnapshot()
})
