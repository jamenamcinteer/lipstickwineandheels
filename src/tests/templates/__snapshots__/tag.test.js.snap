// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render tag template correctly 1`] = `
<PageTransition>
  <div
    style={
      Object {
        "background": "#fff",
      }
    }
  >
    <HelmetWrapper
      defer={true}
      encodeSpecialCharacters={true}
      title="Tech | Lipstick, Wine, and Heels"
    >
      <html
        lang="en"
      />
      <meta
        content="Lipstick, Wine, and Heels is a lifestyle blog dedicated to chic living and nerdy topics. I am a woman in tech who appreciates fashion, fine food and drink, crafty hobbies, fitness, and cats."
        name="description"
      />
    </HelmetWrapper>
    <div
      className="heroContainer"
    >
      <div
        className="hero"
        style={
          Object {
            "backgroundImage": "url(/reference/to/some/image.jpg)",
          }
        }
      />
    </div>
    <div
      className="tagContainer"
    >
      <h1
        className="section-headline"
      >
        Recent articles in 
        Tech
      </h1>
      <ul
        className="article-list"
      >
        <li
          key="this-is-a-sample-blog-post"
        >
          <Component
            article={
              Object {
                "date": "August 30, 2018",
                "slug": "this-is-a-sample-blog-post",
                "tags": Array [
                  Object {
                    "slug": "tech",
                    "title": "Tech",
                  },
                ],
                "title": "This is a sample blog post",
              }
            }
            postStyles={
              Object {
                "gridTemplateColumns": "100%",
                "margin": "0 auto",
                "maxWidth": "660px",
              }
            }
          />
        </li>
        <li
          key="hello-world"
        >
          <Component
            article={
              Object {
                "date": "August 28, 2018",
                "slug": "hello-world",
                "tags": Array [
                  Object {
                    "slug": "tech",
                    "title": "Tech",
                  },
                  Object {
                    "slug": "travel",
                    "title": "Travel",
                  },
                ],
                "title": "Hello World!",
              }
            }
            postStyles={
              Object {
                "gridTemplateColumns": "100%",
                "margin": "0 auto",
                "maxWidth": "660px",
              }
            }
          />
        </li>
      </ul>
    </div>
  </div>
</PageTransition>
`;
